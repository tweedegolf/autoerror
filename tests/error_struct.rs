use autoerror::AutoError;

#[derive(Debug, AutoError)]
struct Error {
    A: std::io::Error,
    B: std::fmt::Error,
}

pub fn main() {
}
