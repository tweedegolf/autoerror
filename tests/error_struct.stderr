error: AutoError only supports enums
 --> $DIR/error_struct.rs:4:8
  |
4 | struct Error {
  |        ^^^^^
